slam-libraries/viam-orb-slam3/orbslam_server_v1.cc

```diff
@@ -498,22 +484,10 @@ void SLAMServiceImpl::ProcessDataOffline(ORB_SLAM3::System *SLAM) {
                 BOOST_LOG_TRIVIAL(fatal) << "Invalid slam_mode=" << slam_mode;
             }
 
-            // Update the copy of the current map whenever a change in
-            // keyframes occurs
-            ORB_SLAM3::Map *currMap = SLAM->GetAtlas()->GetCurrentMap();
-            std::vector<ORB_SLAM3::KeyFrame *> keyframes =
-                currMap->GetAllKeyFrames();
-            {
-                std::lock_guard<std::mutex> lock(slam_mutex);
-                if (SLAM->GetTrackingState() ==
-                    ORB_SLAM3::Tracking::eTrackingState::OK) {
-                    poseGrpc = tmpPose.inverse();
-                    if (nkeyframes != keyframes.size()) {
-                        currMapPoints = currMap->GetAllMapPoints();
-                    }
-                }
-            }
-            nkeyframes = keyframes.size();
+            UpdateMapAndPosition(SLAM, tmpPose);
+
+            // This log line is needed by rdk integration tests.
```

Comment:
added this log and print for consistency between offline/online, however its not currently being used in integration tests in offline mode

Reply:
Fine with removing the log, as we have the `Passing image to SLAM` log already. Integration tests for offline mode are already setup and only look to see if all images are completed currently